scripts/bundleSize/getBundleData.js

```diff
@@ -1,13 +1,13 @@
-const fs = require('fs');
-const { extractBundlesForPageType } = require('./pageTypeBundleExtractor');
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable import/extensions */
```

Comment:
Can we remove this (as it appears we are using the extensions on imports from local files)